vince-fugnitto/local-history-ext
# Complete Story 1

Completes: #1 

#### What it does
1. create local history quick-pick command, and editor context-item
2. quick-pick displays 3 placeholders (will later be replaced by 10 timestamps).

#### How to test
- Press F5 to run the tests in a new window with your extension loaded.
- Create a new file
- Open the Command Palette (Ctrl+Shift+P), enter `Local History: View All`
- Right click in the active editor and sellect `Local History: View All`
- Should be able to see the 3 placeholders

Signed-off-by: Kaiyue Pan <kaiyue.pan@ericsson.com>
